package.json

```diff
@@ -8,7 +8,7 @@
     "node": ">=v16.13.1"
   },
   "scripts": {
-    "storybook": "start-storybook -p 6006 -c storybook",
+    "storybook": "IS_DEV=true start-storybook -p 6006 -c storybook",
```

Comment:
For context for anyone else looking at this, this is to allow us to use this environment variable for local dev to see the brand image asset